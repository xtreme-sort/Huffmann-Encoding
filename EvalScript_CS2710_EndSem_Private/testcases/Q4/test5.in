GenerateLengthLimitedHuffman
qvbjtqhiezrnfcfrruvihripihvgxzxmbfyjifbigeiztwizdsjpjdtehhclfqnpxavxjnuysfopxqhtlnrqoucxafkfqalvemcjdtwmfxdgjszsukqltkftciwbhjvymcdtkxbhgxistbfqwnueyulkrfczmnpprjjuaberkktahmjrzszzqnexlkestzwmerhbbafyytiwgszcporsudetnvjmxyj
8
Quit
