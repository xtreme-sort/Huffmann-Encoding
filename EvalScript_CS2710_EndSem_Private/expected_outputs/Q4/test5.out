f 0000
i 0001
j 0010
r 0011
t 0100
x 0101
z 0110
a 01110
b 01111
c 10000
d 10001
e 10010
h 10011
k 10100
l 10101
m 10110
n 10111
p 11000
q 11001
s 11010
u 11011
v 11100
w 11101
y 11110
g 111110
o 111111
cost: 1040
